ports/cef/window.rs

```diff
@@ -432,12 +429,10 @@ impl WindowMethods for Window {
            check_ptr_exist!(browser.get_host().get_client().get_display_handler(), on_title_change) {
             browser.get_host().get_client().get_display_handler().on_title_change((*browser).clone(), str.as_slice());
         }
-        match &mut *title_visitor {
-            &mut None => {},
-            &mut Some(ref mut visitor) => {
-                visitor.visit(&str);
-            }
-        };
+
+        if let Some(ref mut visitor) = &mut *title_visitor {
```

Comment:
Oops. Missed this. You can remove the `&mut`, and ensure that the build succeeds when we run `./mach build-cef` (it's specific to this file).